### ISSUE TITLE
Upgrading from V2 to V3 definitions/methods/namespace missing

### ISSUE THREAD

**Issue by jiecerbo** [bcemmett/SurveyMonkeyApi-v3#45]
After installing V3 to an existing app, i get the error for the following definitions/methods/namespace:

Respondent
CustomVariableAnswer
RespondentId
QuestionId
FillMissingSurveyInformation
GetRespondentList

**Comment by bcemmett** [comment]
Yes, I'm afraid that pretty much everything about their api changed, including things like properties / names of endpoints. The new library is completely rewritten from v2 -> v3 to match their new api, so everything has changed. Most v2 things have some kind of equivalent in the v3 library, but I gave priority to naming things to match SurveyMonkey's api over matching the v2 library.

**Comment by jiecerbo** [comment]
What would be the equivalent CustomVariableAnswer in v3? I can't seem to find it.
